people/ondrej-bojar.md

```diff
@@ -0,0 +1,12 @@
+---
+grand_parent: Community
+parent: People
+title: Ondřej Bojar
+description: Machine translation researcher
+---
+
+**Ondřej Bojar** is a researcher and associate professor at the [Institute of Formal and Applied Linguistics](https://ufal.mff.cuni.cz/) at Charles University in Prague, widely cited for his work on statistical and later [neural machine translation](/neural-machine-translation).
+He co-authored Moses, a kit for [statistical machine translation](/statistical-machine-translation) and is a long-term organizer of [WMT](/community/communities.md#workshop_on_statistical_machine_translation).
```

Comment:
I guess if we link `statistical machine translation` in the previous paragraph, we can avoid it here.

We have a new page for **WMT**, which was reviewed by Barry Haddow!😄 You can link **WMT** to `/events/wmt.md`.

Reply:
I overlooked that, thanks. :-)